hack/lib/strimzi.bash

```diff
@@ -4,18 +4,17 @@ function install_strimzi_operator {
   strimzi_version=$(curl https://github.com/strimzi/strimzi-kafka-operator/releases/latest |  awk -F 'tag/' '{print $2}' | awk -F '"' '{print $1}' 2>/dev/null)
   header "Installing Strimzi Kafka operator"
   oc create namespace kafka
-  oc -n kafka apply --selector strimzi.io/crd-install=true -f "https://github.com/strimzi/strimzi-kafka-operator/releases/download/${strimzi_version}/strimzi-cluster-operator-${strimzi_version}.yaml"
   curl -L "https://github.com/strimzi/strimzi-kafka-operator/releases/download/${strimzi_version}/strimzi-cluster-operator-${strimzi_version}.yaml" \
   | sed 's/namespace: .*/namespace: kafka/' \
-  | oc -n kafka apply -f -
+  | oc -n kafka create -f -
```

Comment:
Why replacing `apply` with `create`? Those scripts should be safe to run multiple times. Create will fail for second run.

Reply:
`apply` cannot handle big CRDs.